packages/core/src/components/slider/slider.tsx

```diff
@@ -0,0 +1,64 @@
+import { Component, h, Prop } from "@stencil/core";
+
+@Component({
+  tag: "fl-slider",
+  styleUrl: "slider.scss",
+  shadow: true,
+})
+export class Slider {
+  constructor() {
+    this.sliderBar = this.sliderBar.bind(this);
+  }
+  @Prop() value = 50;
+
+  @Prop() min = 0;
+
+  @Prop() max = 100;
+
+  @Prop() disabled: boolean;
+
+  private sliderRef?: HTMLInputElement;
+  private fillRef?: HTMLSpanElement;
+
+  sliderBar() {
+    this.fillRef.style.width =
+      ((+this.sliderRef.value - this.min) * 100) / (this.max - this.min) + "%";
+  }
```

Comment:
If you use an arrow function, you don't need to `bind` the function.

```suggestion
  const sliderBar = () => {
    this.fillRef.style.width =
      ((+this.sliderRef.value - this.min) * 100) / (this.max - this.min) + "%";
  }
```